src/TeachABit.Model/DTOs/Tecaj/TecajDto.cs

```diff
@@ -0,0 +1,6 @@
+namespace TeachABit.Model.DTOs.Tecaj;
+
+public class TecajDto
+{
+    public string Naziv {get; set;}
```

Comment:
Ovdje dodaj "Id" polje, zato što će na frontu trebati dohvaćati specifične tečaje po njihovom Id-u.
Također postavi default vrijednost za Naziv da je string.Empty.